addons/mail/views/mail_canned_response_views.xml

```diff
@@ -61,12 +61,10 @@
             }</field>
             <field name="help" type="html">
                 <p class="o_view_nocontent_smiling_face">
-                    Create a new canned response
+                    No canned response found. Let’s create one!
                 </p><p>
-                    Canned responses allow you to insert prewritten responses in
-                    your messages by typing <i>:shortcut</i>. The shortcut is
-                    replaced directly in your message, so that you can still edit
-                    it before sending.
+                    Use canned responses to quickly insert prewritten messages
+                    with <i>::shortcut</i>. You can edit the text before sending.
```

Comment:
This should be a single line, it would make the translation much less confusing (many extra spaces and unnecessary \n in the middle).